test/init_psql/initdb_citext.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
when and how will this script get called? I assume the mounting `./test/init_psql:/docker-entrypoint-initdb.d` will take care of it automatically?

Reply:
yes - see https://hub.docker.com/_/postgres `Initialization scripts` section